Documentation/Contributing.md

```diff
@@ -0,0 +1,23 @@
+Please always start your contribution with an Issue, per [.NET Core Contributions](https://github.com/dotnet/corefx/blob/master/Documentation/contributing.md). The recommended contribution workflow is described in those guidelines.
```

Comment:
Suggestion, but not required.  CoreFx restructured theirs a bit to be shorter and have a certain style.  Example:  theirs is [Contributing](https://github.com/dotnet/corefx/blob/master/Documentation/contributing.md).  If you see value in following their style, consider it, otherwise ignore this comment